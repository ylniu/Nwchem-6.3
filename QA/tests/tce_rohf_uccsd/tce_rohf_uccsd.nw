#
# Water ROHF-UCCSD/STO-3G
#
# ROHF-UCCSD   -0.0412114
#
start tce_rohf_uccsd

echo

charge 1

geometry units angstrom
zmat
H
O H 0.96
H O 0.96 H 104.0
end
end

basis
* library sto-3g
end

scf
thresh 1e-10
tol2e 1e-10
doublet
rohf
end

tce
ccsd
end

task tce energy
