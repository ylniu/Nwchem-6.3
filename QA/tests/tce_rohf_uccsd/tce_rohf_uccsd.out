 
Processor list
 
cu04n[119,121]
 
ARMCI configured for 2 cluster nodes. Network protocol is 'OpenIB Verbs API'.
All connections between all procs tested: SUCCESS
 argument  1 = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/tce_rohf_uccsd/tce_rohf_uccsd.nw



============================== echo of input deck ==============================
#
# Water ROHF-UCCSD/STO-3G
#
# ROHF-UCCSD   -0.0412114
#
start tce_rohf_uccsd

echo

charge 1

geometry units angstrom
zmat
H
O H 0.96
H O 0.96 H 104.0
end
end

basis
* library sto-3g
end

scf
thresh 1e-10
tol2e 1e-10
doublet
rohf
end

tce
ccsd
end

task tce energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = cu4n119
    program       = /scratch/nwchem
    date          = Fri Oct 29 11:42:30 2010

    compiled      = Thu_Oct_28_07:10:53_2010
    source        = /home/scicons/user/kurt/nwchem-6.0-release-pgf90-final/
    nwchem branch = 6.0
    input         = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/tce_rohf_uccsd/tce_rohf_uccsd.nw
    prefix        = tce_rohf_uccsd.
    data base     = ./tce_rohf_uccsd.db
    status        = startup
    nproc         =        8
    time left     =   1764s



           Memory information
           ------------------

    heap     =  104857601 doubles =    800.0 Mbytes
    stack    =  104857601 doubles =    800.0 Mbytes
    global   =  209715200 doubles =   1600.0 Mbytes (distinct from heap & stack)
    total    =  419430402 doubles =   3200.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 C2V symmetry detected


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 H                    1.0000     0.75649032     0.00000000    -0.47282801
    2 O                    8.0000     0.00000000     0.00000000     0.11820700
    3 H                    1.0000    -0.75649032     0.00000000    -0.47282801

      Atomic Mass 
      ----------- 

      H                  1.007825
      O                 15.994910


 Effective nuclear repulsion energy (a.u.)       9.1693789250

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             C2v       
 Group number             16
 Group order               4
 No. of unique centers     2

      Symmetry unique atoms

     1    2



                                Z-matrix (user)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  2     1                       0.96000
    2 Stretch                  3     2                       0.96000
    3 Bend                     3     2     1               104.00000


            XYZ format geometry
            -------------------
     3
 geometry
 H                     0.75649032     0.00000000    -0.47282801
 O                     0.00000000     0.00000000     0.11820700
 H                    -0.75649032     0.00000000    -0.47282801

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 O                |   1 H                |     1.81414  |     0.96000
    3 H                |   2 O                |     1.81414  |     0.96000
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    1 H                |   2 O                |   3 H                |   104.00
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: <
 /mscf/scicons/apps/nwchem-6.0.oct19//src/basis/libraries/>
  


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           sto-3g                   on all atoms 


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.42525091E+00  0.154329
  1 S  6.23913730E-01  0.535328
  1 S  1.68855400E-01  0.444635

  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.30709320E+02  0.154329
  1 S  2.38088610E+01  0.535328
  1 S  6.44360830E+00  0.444635

  2 S  5.03315130E+00 -0.099967
  2 S  1.16959610E+00  0.399513
  2 S  3.80389000E-01  0.700115

  3 P  5.03315130E+00  0.155916
  3 P  1.16959610E+00  0.607684
  3 P  3.80389000E-01  0.391957



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                           sto-3g                  1        1   1s
 O                           sto-3g                  3        5   2s1p


                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =     7
  atoms           =     3
  closed shells   =     4
  open shells     =     1
  charge          =   1.00
  wavefunction    = ROHF
  input vectors   = atomic
  output vectors  = ./tce_rohf_uccsd.movecs
  use symmetry    = T
  symmetry adapt  = T


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                           sto-3g                  1        1   1s
 O                           sto-3g                  3        5   2s1p


      Symmetry analysis of basis
      --------------------------

        a1          4
        a2          0
        b1          2
        b2          1


 Forming initial guess at       1.7s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -74.71095592

 Renormalizing density from      10.00 to      9

      Non-variational initial energy
      ------------------------------

 Total energy =     -69.703127
 1-e energy   =    -109.566527
 2-e energy   =      30.694021
 HOMO         =      -0.977577
 LUMO         =      -0.045178


      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------

  Numbering of irreducible representations: 

     1 a1          2 a2          3 b1          4 b2      

  Orbital symmetries:

     1 a1          2 a1          3 b1          4 a1          5 b2      
     6 a1          7 b1      


 Starting SCF solution at       1.8s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-10
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-10
 ----------------------------------------------


 #quartets = 7.900D+01 #integrals = 1.640D+02 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_rohf_uccsd.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 169376
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -74.6111008567  6.98D-01  5.79D-01      0.8
                 2      -74.6537066435  1.80D-01  1.59D-01      0.8
                 3      -74.6544061201  2.97D-02  2.41D-02      0.8
                 4      -74.6544568273  3.58D-04  2.60D-04      0.8
                 5      -74.6544568462  1.15D-08  9.63D-09      0.9
                 6      -74.6544568462  1.26D-12  1.01D-12      1.0


       Final ROHF results 
       ------------------ 

         Total SCF energy =    -74.654456846227
      One-electron energy =   -116.395106712262
      Two-electron energy =     32.571270941050
 Nuclear repulsion energy =      9.169378924985

        Time for solution =      0.2s



       Symmetry analysis of molecular orbitals - final
       -----------------------------------------------

  Numbering of irreducible representations: 

     1 a1          2 a2          3 b1          4 b2      

  Orbital symmetries:

     1 a1          2 a1          3 b1          4 a1          5 b2      
     6 a1          7 b1      

             Final eigenvalues
             -----------------

              1      
    1  -21.0168
    2   -1.8402
    3   -1.1760
    4   -1.0383
    5   -0.6698
    6    0.0994
    7    0.2057

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    2  Occ=2.000000D+00  E=-1.840247D+00  Symmetry=a1
              MO Center= -3.5D-18, -7.6D-38, -6.7D-02, r^2= 4.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.884444  2 O  s                  2     -0.239354  2 O  s          
     6     -0.175728  2 O  pz         

 Vector    3  Occ=2.000000D+00  E=-1.175971D+00  Symmetry=b1
              MO Center=  7.8D-16, -2.6D-30, -6.5D-02, r^2= 7.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.720331  2 O  px                 1      0.342727  1 H  s          
     7     -0.342727  3 H  s          

 Vector    4  Occ=2.000000D+00  E=-1.038297D+00  Symmetry=a1
              MO Center= -1.5D-15,  2.9D-21,  2.1D-01, r^2= 5.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.847601  2 O  pz                 3      0.449197  2 O  s          
     7     -0.192718  3 H  s                  1     -0.192718  1 H  s          

 Vector    5  Occ=1.000000D+00  E=-6.698481D-01  Symmetry=b2
              MO Center= -2.0D-32, -1.5D-18,  1.2D-01, r^2= 4.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      1.000000  2 O  py         

 Vector    6  Occ=0.000000D+00  E= 9.941979D-02  Symmetry=a1
              MO Center=  7.2D-16,  1.5D-18, -4.5D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.879362  2 O  s                  1     -0.827694  1 H  s          
     7     -0.827694  3 H  s                  6     -0.649794  2 O  pz         

 Vector    7  Occ=0.000000D+00  E= 2.057242D-01  Symmetry=b1
              MO Center= -1.7D-16, -1.2D-32, -3.2D-01, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.908181  2 O  px                 7      0.883614  3 H  s          
     1     -0.883614  1 H  s          


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.09838174

 moments of inertia (a.u.)
 ------------------
           2.233026115202           0.000000000000           0.000000000000
           0.000000000000           6.352289379065           0.000000000000
           0.000000000000           0.000000000000           4.119263263863

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 H    1     0.55   0.55
    2 O    8     7.90   2.00  1.88  4.02
    3 H    1     0.55   0.55

 Mulliken analysis of the open-shell density
 -------------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 H    1     0.00   0.00
    2 O    8     1.00   0.00  0.00  1.00
    3 H    1     0.00   0.00

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      1.000000     -1.000000     10.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1     -0.974340     -0.223379      0.000000

     2   2 0 0     -1.795895     -0.297091      4.087280
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -3.477920     -0.891272      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -3.131948     -0.346989      1.995924


 Parallel integral file used       8 records with       0 large values

                   NWChem Extensible Many-Electron Theory Module
                   ---------------------------------------------

              ======================================================
                   This portion of the program was automatically
                  generated by a Tensor Contraction Engine (TCE).
                  The development of this portion of the program
                 and TCE was supported by US Department of Energy,
                Office of Science, Office of Basic Energy Science.
                      TCE is a product of Battelle and PNNL.
              Please cite: S.Hirata, J.Phys.Chem.A 107, 9887 (2003).
              ======================================================

            General Information
            -------------------
      Number of processors :     8
         Wavefunction type : Restricted open-shell Hartree-Fock
          No. of electrons :     9
           Alpha electrons :     5
            Beta electrons :     4
           No. of orbitals :    14
            Alpha orbitals :     7
             Beta orbitals :     7
        Alpha frozen cores :     0
         Beta frozen cores :     0
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity : doublet 
    Number of AO functions :     7
       Number of AO shells :     5
        Use of symmetry is : on 
      Symmetry adaption is : on 
         Schwarz screening : 0.10D-09

          Correlation Information
          -----------------------
          Calculation type : Coupled-cluster singles & doubles                           
   Perturbative correction : none                                                        
            Max iterations :      100
        Residual threshold : 0.10D-06
     T(0) DIIS level shift : 0.00D+00
     L(0) DIIS level shift : 0.00D+00
     T(1) DIIS level shift : 0.00D+00
     L(1) DIIS level shift : 0.00D+00
     T(R) DIIS level shift : 0.00D+00
     T(I) DIIS level shift : 0.00D+00
   CC-T/L Amplitude update :  5-th order DIIS
                I/O scheme : Global Array Library

            Memory Information
            ------------------
          Available GA space size is    1677721502 doubles
          Available MA space size is     209714465 doubles

 Maximum block size        76 doubles

 tile_dim =      3

 Block   Spin    Irrep     Size     Offset   Alpha
 -------------------------------------------------
   1    alpha     a1     3 doubles       0       1
   2    alpha     b1     1 doubles       3       2
   3    alpha     b2     1 doubles       4       3
   4    beta      a1     3 doubles       5       4
   5    beta      b1     1 doubles       8       5
   6    alpha     a1     1 doubles       9       6
   7    alpha     b1     1 doubles      10       7
   8    beta      a1     1 doubles      11       8
   9    beta      b1     1 doubles      12       9
  10    beta      b2     1 doubles      13      10

 Global array virtual files algorithm will be used

 Parallel file system coherency ......... OK

 #quartets = 1.200D+02 #integrals = 2.560D+02 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_rohf_uccsd.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 169376
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


 Fock matrix recomputed
 1-e file size   =               42
 1-e file name   = ./tce_rohf_uccsd.f1 
 Cpu & wall time / sec            0.0            0.1

 tce_ao2e: fast2e=1
 half-transformed integrals in memory

 2-e (intermediate) file size =            5439
 2-e (intermediate) file name = ./tce_rohf_uccsd.v2i
 Cpu & wall time / sec            0.0            0.0

 tce_mo2e: fast2e=1
 2-e integrals stored in memory

 2-e file size   =             1523
 2-e file name   = ./tce_rohf_uccsd.v2 
 Cpu & wall time / sec            0.0            0.0
 do_pt =   F
 do_lam_pt =   F
 do_cr_pt =   F
 do_lcr_pt =   F
 do_2t_pt =   F
 T1-number-of-tasks                        4

 t1 file size   =                8
 t1 file name   = ./tce_rohf_uccsd.t1 
 t1 file handle =       -998
 T2-number-of-boxes                       24

 t2 file size   =               94
 t2 file name   = ./tce_rohf_uccsd.t2 
 t2 file handle =       -995

 CCSD iterations
 -----------------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall    V2*C2
 -----------------------------------------------------------------
    1   0.0776942569601  -0.0297769582804     0.1     0.1     0.0
    2   0.0253585960998  -0.0376250943559     0.1     0.1     0.0
    3   0.0097662347532  -0.0398962335395     0.1     0.1     0.0
    4   0.0039984530662  -0.0406969817010     0.1     0.1     0.0
    5   0.0016965170410  -0.0410028204467     0.1     0.1     0.0
 MICROCYCLE DIIS UPDATE:                        5                        5
    6   0.0000562319879  -0.0412103935232     0.1     0.1     0.0
    7   0.0000185850787  -0.0412109060956     0.1     0.1     0.0
    8   0.0000083534666  -0.0412112384483     0.1     0.1     0.0
    9   0.0000040087674  -0.0412113601180     0.1     0.1     0.0
   10   0.0000019679753  -0.0412114066491     0.1     0.1     0.0
 MICROCYCLE DIIS UPDATE:                       10                        5
   11   0.0000000901581  -0.0412114412467     0.1     0.1     0.0
 -----------------------------------------------------------------
 Iterations converged
 CCSD correlation energy / hartree =        -0.041211441246700
 CCSD total energy / hartree       =       -74.695668287473620

 Singles contributions

 Doubles contributions

 Parallel integral file used       8 records with       0 large values


 Task  times  cpu:        1.2s     wall:        1.7s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  758      758     5116      848     2056        0        0        0     
number of processes/call 1.16e+00 1.13e+00 1.19e+00 0.00e+00 0.00e+00
bytes total:             3.65e+05 1.46e+05 1.42e+05 0.00e+00 0.00e+00 0.00e+00
bytes remote:            1.02e+05 1.42e+04 3.74e+04 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 7800 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        18	        32
	current total bytes		         0	         0
	maximum total bytes		   1053752	  22509016
	maximum total K-bytes		      1054	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        1.4s     wall:        3.4s
